libs/transloco/src/lib/helpers.ts

```diff
@@ -86,10 +86,14 @@ export function coerceArray<T>(value: T | T[]): T[] {
  * given: path-to_happiness => pathToHappiness
  *
  */
-export function toCamelCase(str: string): string {
+export function toCamelCase(str: string, ignoreLowerUpperCase = false): string {
```

Comment:
Here I was a bit in doubt myself to be honest.

Because this method does not only do a camel casing but also replaces, dashes, underscorse, forward slashes, spaces... so i was a bit confused.

So I added this to comply with the fact we only ignore the casing, but I think it would indeed be better to just let it up to the consumer.

But then the next question might rise, is `keepNamespaceCasing` still a good name?
Maybe we could go for `scope.keepNamespace` of `scope.noCamelCase` ?

Reply:
I find `keepNamespaceCasing` very explicit. Maybe `scope.keepCasing`.

Reply:
Agreed, changed